src/DirectAdmin/DirectAdmin.php

```diff
@@ -101,8 +101,6 @@ protected function __construct($url, $username, $password)
         $this->connection = new Client([
             'base_uri' => $this->baseUrl,
             'auth' => [$username, $password],
-            'http_errors' => true,
```

Comment:
If the DA API returns those 4xx/5xx errors for valid situations, shouldn't this be set to `false`, i.e. don't throw an exception when receiving those status codes?

I removed that line as the default from guzzle is `true`, so setting it explicitly to `true` doesn't change anything. See http://docs.guzzlephp.org/en/latest/request-options.html?highlight=http_errors#http-errors